src/pages/contact.tsx

```diff
@@ -10,9 +10,7 @@ import Hero from '../components/Hero';
 export default function Contact() {
   return (
     <>
-      {/* <Seo templateTitle='Home' /> */}
-      {/*  */}
-      <Seo />
+      <Seo templateTitle="Contact" />
 
       <main>
         <section className="">
```

Comment:
Please confirm that the form on this page is valid and functional. Please add a section to the doc specifying which email should be notified when this form is submitted.